products/tomcat.md

```diff
@@ -26,16 +26,16 @@ releases:
     latestReleaseDate: 2022-10-03
 -   releaseCycle: "9"
     eol: false
-    latest: "9.0.71"
+    latest: "9.0.72"
     link: https://tomcat.apache.org/download-90.cgi
     releaseDate: 2017-09-27
-    latestReleaseDate: 2023-01-09
+    latestReleaseDate: 2023-02-18
 -   releaseCycle: "8.5"
     eol: 2024-03-31
-    latest: "8.5.85"
+    latest: "8.5.86"
```

Comment:
Those dates are consistent with https://github.com/apache/tomcat/tags, but it is on only updated here when releases are available on Maven Central. So LGTM.